packages/plugin-page-view-tracking-browser/src/page-view-tracking.ts

```diff
@@ -48,46 +81,48 @@ export const pageViewTrackingPlugin = (
         attributionConfig.trackPageViews = false;
       }
 
-      if (
-        typeof pageTrackingConfig.trackOn === 'undefined' ||
-        (typeof pageTrackingConfig.trackOn === 'function' && pageTrackingConfig.trackOn())
-      ) {
-        const event = createPageViewEvent();
-        event.event_properties = {
-          ...(await getCampaignParamsForPageViewEvent()),
-          ...event.event_properties,
-        };
-        instance.track(event);
+      if (shouldTrackOnPageLoad()) {
+        instance.track(await createPageViewEvent());
+      }
+
+      /* istanbul ignore next */
+      if (pageTrackingConfig.trackHistoryChanges) {
```

Comment:
curious if we truly need this, or if we can just treat SPA and non-SPA page transitions as the same

Reply:
Like - why don't we track SPA page views by default?

Reply:
Yeah

Reply:
Other MA tools don't do it by default, so it's unexpected. Many customers considering switching to amplitude for marketing analytics want to see data parity with GA/Adobe before switching. Also replacing the global pushState is a bit of a hack, so I'm hesitant to do it by default.